src/jsd/actuator_fsm_helpers.cc

```diff
@@ -205,7 +205,10 @@ bool fastcat::Actuator::HandleNewProfPosCmd(DeviceCmd& cmd)
 
   // Only transition to disengaging if its needed
   if(state_->actuator_state.servo_enabled){
-    MSG("Bypassing wait since brakes are disengaged");
+    // Only print this if we are not already in a position profile
+    if(actuator_sms_ != ACTUATOR_SMS_PROF_POS){
+      MSG("Bypassing wait since brakes are disengaged");
```

Comment:
I don't think these will ever print. I believe you have to be in the corresponding state machine state for these functions to execute.

In my opinion, these print messages should be deleted altogether since we are in the real-time loop and it is not an error message.